apps/leave-requests/lib/anniversary-utils.ts

```
@@ -0,0 +1,284 @@
+  const thisMonthAnniversaries = [];
+  
+  for (const user of users) {
+    const start = new Date(user.start_date);
+    const anniversaryThisYear = new Date(currentYear, start.getMonth(), start.getDate());
+    
+    // Check if anniversary is in current month
+    if (anniversaryThisYear.getMonth() === currentMonth) {
+      const daysUntil = Math.ceil((anniversaryThisYear.getTime() - today.getTime()) / (1000 * 60 * 60 * 24));
+      
+      // Include all anniversaries in the current month (including passed ones)
+      if (true) { // Show all anniversaries in current month
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)